vl-grid.src.js

```diff
@@ -140,6 +150,14 @@ export class VlColumn extends NativeVlElement(HTMLDivElement) {
     this._changeClass(this, (this._smallSizeAttribute + '-' + oldValue + '--s'), (this._smallSizeAttribute + '-' + newValue + '--s'), this._classPrefix);
   }
 
+  _extra_small_sizeChangedCallback(oldValue, newValue) {
```

Comment:
Aangezien hier een bug in zat en dit niet bleek uit de unit testen misschien ook een unit test voorzien die de sizes aanpast?